src/dnscap.1.in

```diff
@@ -353,6 +354,11 @@ Enable immediate mode on interfaces.
 Append "and
 .Ar str
 " to the pcap filter.
+.It Fl q Ar unsigned int
+Causes the output to be sampled after the application of all other filters.
+Only every nth dns initiation will be output and responses are only output
+if they correspond to one of the sampled queries. This option cannot be
+used with option.
```

Comment:
I'm very sorry, this must have been a copy and paste error. The last sentence is only half of the sentence it should be, I'll correct that.